docs/csv-description.md

```diff
@@ -220,20 +220,20 @@ If the `koku-metrics-operator` is configured to run in a restricted network, the
   $ oc delete -f volume-shell.yaml
   ```
 
-## Create a Source
-In a restricted network, the `koku-metrics-operator` cannot automatically create a source. This process must be done manually. In the console.redhat.com platform, open the [Sources menu](https://console.redhat.com/settings/sources/) to begin adding an OpenShift source to Cost Management:
+## Create an Integration
+In a restricted network, the `koku-metrics-operator` cannot automatically create an integration. This process must be done manually. In the console.redhat.com platform, open the [Integrations menu](https://console.redhat.com/settings/integrations/) to begin adding an OpenShift integration to Cost Management:
 
 Prerequisites:
 * The cluster identifier which can be found in the KokuMetricsConfig CR, the cluster Overview page, or the cluster Help > About.
 
-Creating a source:
+Creating an integration:
 1. Navigate to the Sources menu
 2. Select the `Red Hat` tab
-3. Create a new `Red Hat Openshift Container Platform` source:
-    * give the source a unique name
+3. Create a new `Red Hat Openshift Container Platform` integration:
+    * give the integration a unique name
     * add the Cost Management application
     * add the cluster identifier
-4. In the Source wizard, review the details and click `Finish` to create the source.
+4. In the Integration wizard, review the details and click `Finish` to create the integration.
```

Comment:
On Nov 1, I believe this will still be called the Add Red Hat source. Then, in phase 2, which hasn't been scheduled, it will change to Add Red Hat integration.

Reply:
Update! Thanks for the 👀 

<img src="https://media4.giphy.com/media/2OHYMJGnGoiVjVg55P/giphy.gif"/>